<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConvert</class>
 <widget class="QDialog" name="frmConvert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QNapi - Convert subtitles</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resources.qrc">
    <normaloff>:/icon/qnapi.png</normaloff>:/icon/qnapi.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbSrcSubFile">
     <property name="text">
      <string>Source subtitles file:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="leSrcSubFile">
     <property name="text">
      <string/>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pbSrcSubFileSelect">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res/resources.qrc">
       <normaloff>:/ui/folder.png</normaloff>:/ui/folder.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QLabel" name="lbDetectedFormat">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Detected format:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbDetectedFormatValue">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>280</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="lbTargetFormat">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Target subtitles format:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cbTargetFormat">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>180</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentText">
        <string notr="true">mDVD</string>
       </property>
       <item>
        <property name="text">
         <string notr="true">mDVD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">MPL2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SRT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">TMP</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="lbMovieFPS">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Video framerate (FPS):</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cbMovieFPS">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>23.976</string>
       </property>
       <item>
        <property name="text">
         <string>23.976</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>29.97</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>59.94</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>60</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pbMovieFPSSelect">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Detect video framerate (FPS) form file</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res/resources.qrc">
       <normaloff>:/ui/folder.png</normaloff>:/ui/folder.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lbTargetExtension">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>File extension for target subtitles file:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cbTargetExtension">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Default</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>srt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sub</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>txt</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cbChangeFPS">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Change FPS (frames/second) of subtitles</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbFrom">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>from</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbFPSFrom">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>23.976</string>
       </property>
       <item>
        <property name="text">
         <string>23.976</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>29.97</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>59.94</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>60</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbTo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>to</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbFPSTo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>23.976</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>29.97</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>59.94</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>60</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pbTargetMovieFPSSelect">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Detect video framerate (FPS) form file</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res/resources.qrc">
       <normaloff>:/ui/folder.png</normaloff>:/ui/folder.png</iconset>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QCheckBox" name="cbDelaySubtitles">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Shift subtitles</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sbDelayOffset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="suffix">
        <string> seconds</string>
       </property>
       <property name="minimum">
        <double>-999999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lbTargetFileName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Target subtitles file name:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLineEdit" name="leTargetFileName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>478</width>
       <height>34</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbConvert">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Convert</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbChangeFPS</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbFrom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbChangeFPS</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbFPSFrom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbChangeFPS</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbTo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbChangeFPS</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbFPSTo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbChangeFPS</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbTargetMovieFPSSelect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbDelaySubtitles</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbDelayOffset</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
